projects/demo/src/modules/components/drawer/index.html

```diff
@@ -13,5 +13,114 @@
+            >
+                <header>
+                    <h2 tuiHeader>
+                        <div tuiTitle>Header</div>
+                        <div tuiAccessories>
+                            <button
+                                tuiButton
+                                type="button"
+                                (click)="onClose()"
+                            >
+                                Close
+                            </button>
+                        </div>
+                    </h2>
+                </header>
+
+                <p>
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Cupiditate perspiciatis exercitationem
+                    nemo velit aliquam voluptates non porro, vel, nihil laudantium sapiente ex omnis corrupti assumenda
+                    voluptatibus, architecto sequi saepe consectetur ratione qui. Beatae, sapiente explicabo velit
+                    facere repudiandae veniam et soluta quia qui expedita voluptate accusamus dolor adipisci. Illo quia
+                    sint consequatur unde nulla fuga eum officiis, impedit dolorem? Vel itaque temporibus nihil quia?
+                    Provident earum aperiam autem veritatis hic doloremque unde nesciunt accusantium nisi corrupti.
+                </p>
+
+                <footer tuiHeader>
+                    <div tuiTitle>Footer</div>
+                    <button
+                        appearance="secondary"
+                        tuiButton
+                        type="button"
+                        (click)="onClose()"
+                    >
+                        Close
+                    </button>
+                </footer>
+            </tui-drawer>
+        </tui-doc-demo>
+
+        <table tuiDocAPI>
+            <tr
+                name="[direction]"
+                tuiDocAPIItem
+                type="TuiHorizontalDirection"
+                [items]="directionVariants"
+                [(value)]="direction"
+            >
+                Drawer opening direction. Default is
+                <code>'right'</code>
+                .
+            </tr>
+
+            <tr
+                name="[overlay]"
+                tuiDocAPIItem
+                type="boolean"
+                [(value)]="overlay"
+            >
+                Show overlay under the drawer.
+            </tr>
+
+            <tr
+                name="header"
+                tuiDocAPIItem
+                type="TemplateRef"
+            >
+                Slot for custom header content. Use
+                <code>&lt;header&gt;</code>
+                .
+            </tr>
+
+            <tr
+                name="footer"
+                tuiDocAPIItem
+                type="TemplateRef"
+            >
+                Slot for custom footer content. Use
+                <code>&lt;footer&gt;</code>
+                .
+            </tr>
+
+            <tr
+                name="[scrollbar]"
```

Comment:
This one should be removed, there's no projection of scrollbar at the moment. Maybe we will add it later as a fix for:
https://github.com/taiga-family/taiga-ui/issues/9680
But right now there should be no mention of it here.